function_pipe/core/function_pipe.py

```diff
@@ -70,6 +70,38 @@ def binary(lhs: FN, rhs: tp.Any) -> FN:
     return binary
 
 
+def _from_this_module(some_tb: types.TracebackType) -> bool:
+    return some_tb.tb_frame.f_code.co_filename == __file__
```

Comment:
Not sure "some" is a useful descriptor here!
```suggestion
def _from_this_module(tb: types.TracebackType) -> bool:
    return tb.tb_frame.f_code.co_filename == __file__
```